docs/csharp/programming-guide/classes-and-structs/named-and-optional-arguments.md

```diff
@@ -70,13 +70,14 @@ The following code implements the examples from this section along with some add
 
 ## Optional arguments
 
-The definition of a method, constructor, indexer, or delegate can specify its parameters are required or optional. Any call must provide arguments for all required parameters, but can omit arguments for optional parameters.
+The definition of a method, constructor, indexer, or delegate can specify its parameters are required or optional. Any call must provide arguments for all required parameters, but can omit arguments for optional parameters. Additionally, nullable reference types (`T?`) implicitly have `null` as their default value.
```

Comment:
The new sentence emphasizes the cause of the confusion, rather than removing it. We need to make a clear distinction between an optional parameter and the default value for a type.